Source/Scene/MetadataTable.js

```diff
@@ -269,6 +280,29 @@ MetadataTable.prototype.getPropertyTypedArray = function (propertyId) {
   return undefined;
 };
 
+/**
+ * Returns a typed array containing the property values for the property with the given semantic.
+ *
+ * @param {String} semantic The case-sensitive semantic of the property.
+ * @returns {*} The typed array containing the property values or <code>undefined</code> if the property values are not stored in a typed array.
+ *
+ * @private
+ */
+MetadataTable.prototype.getPropertyTypedArrayBySemantic = function (semantic) {
+  //>>includeStart('debug', pragmas.debug);
+  Check.typeOf.string("semantic", semantic);
+  //>>includeEnd('debug');
+
+  if (defined(this._class)) {
+    var property = this._class.propertiesBySemantic[semantic];
```

Comment:
We'd have to be consistent in the naming of the variables too - this would be `this._class.semantics` - i.e. only if we decide one way or the other.